src/site/topic23.rst

```diff
@@ -12,6 +12,37 @@ Topic #23 --- Binary Search Trees
 Binary Search Tree Definition
 =============================
 
+* A binary *search* tree is a binary tree where:
+    * All values in the left subtree are less than the value in the root node
+    * All values in the right subtree are greater than or equal to the value in the root node
+        * We can make the right subtree exclusively greater than, but this would disallow duplicates
```

Comment:
Which comparison gets the equality is arbitrary as well. Using `<=` for the left subtree is also valid, so long as only one side gets the `==`.